### PR TITLE
6.8wip

### PR DESCRIPTION
**Pull request by jeffreytoffoli**
updates for 6.8!

### CODE REVIEW

File: api-guide/inputs/README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # Inputs
 
+Clarifai helps you build AI solutions that process image, video and text inputs. 
+
+
+{% hint style="info" %}
+* 128 is the maximum number of images that can be sent at once
+* Each image should be less than 20MB
+* Max gRPC message size is 10,485,760
```
Comment (yangshengBE) [inline api-guide/inputs/README.md:9]: do we need to mention grpc size?

File: api-guide/workflows/input_nodes.md [hunk 2]
```diff
@@ -31,6 +31,96 @@ You can create workflows out of any Clarifai Models or custom models that you ha
 The The following is an example of how to build a workflow with multiple connected nodes. Note that model IDs and model version IDs from the public `clarifai/main` application are fixed, so they are already hard-coded in the code examples below. It is possible to use other public model or model version IDs.
```
Comment (yangshengBE) [inline api-guide/workflows/input_nodes.md:32]: user can also use their own models in workflow, not only public models

File: api-guide/workflows/input_nodes.md [hunk 3]
```diff
@@ -402,3 +404,207 @@ curl -X POST 'https://api.clarifai.com/v2/users/me/apps/{{app}}/workflows' \
 {% endtab %}
 {% endtabs %}
 
+
+### Suppressing the output from nodes
+
+It is possible to turn the outputs from given nodes in your workflow on and off with the `supress_output` endpoint.
```
Comment (yangshengBE) [inline api-guide/workflows/input_nodes.md:410]: typo, should be `suppress_output`
the default is on, so i think just mention turn it off?
also maybe add the reason why they want to suppress_output?

File: data-labeling-services/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Data Labeling Services
+
+You don't need to hire, train and manage your own labeling team. With Clarifai's Data Labeling Services you can start your project right away and have the work taken care of by Clarifai's dedicated team of expert labelers.
```
Comment (yangshengBE) [inline data-labeling-services/README.md:3]: also mention this service is only available to certain users?

File: portal-guide/data/csv-and-tsv.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# CSV & TSV Uploads
```
Comment (yangshengBE) [inline portal-guide/data/csv-and-tsv.md:1]: what is TSV?

File: api-guide/workflows/input_nodes.md (hunk 3 above)
Comment (jeffreytoffoli) [reply api-guide/workflows/input_nodes.md:410]: Sorry . . . so you're saying we want to state something like "by default, this endpoint will be on when used. Just set the boolean value to false to switch it off."

Comment (yangshengBE) [reply api-guide/workflows/input_nodes.md]: to be clear, the default is `false`, meaning we do not suppress any model's output

File: portal-guide/pinputs.md [hunk 6]
```diff
@@ -28,3 +28,8 @@ You can add metadata to images directly from explorer. Just click on a single im
 
 ![](../.gitbook/assets/fall_collection%20%281%29.jpg)
 
+{% hint style="info" %}
+* 128 is the maximum number of images that can be sent at once
+* Each image should be less than 20MB
+* Max gRPC message size is 10,485,760
```
Comment (yangshengBE) [inline portal-guide/pinputs.md:34]: one more grpc size.

**Review by yangshengBE — APPROVED** [review]

**Pull request merged**